app/Http/Controllers/AuthController.php

```diff
@@ -145,10 +149,4 @@ public function verifySignedRequest($signedRequest)
             return null;
         }
     }
-
-    private function storeToSession(array $configs) {
```

Comment:
Isn't this method still being called in line [103](
https://github.com/findologic/plugin-bigcommerce-di/pull/12/files#diff-138f5876b90f410274432eb9857380a1c69436e2ef1ae6b91702edf334f4ea4aR103)?

Reply:
Method has been moved to parent `app/Http/Controllers/Controller.php`.